src/Services/StudyAreaService.ts

```diff
@@ -73,6 +73,7 @@ module StreamStats.Services {
         extensionDateRange: IDateRange;
         selectedGage: any;        
         flowAnywhereData: any;
+        sedimentModelData: any;

```

Comment:
Create an object (optionally with interface) to keep track of sediment model data from sediment modal (date range and selected gage)